From aa2b75e459d9ab082589d2eae17bff8bd458c1c9 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Sun, 4 Oct 2015 11:14:52 +0000
Subject: [PATCH 0688/1077] MANTIS 0037349 change: integration of ingenico
 payment device for KIS kiosk

massai_systools.lib
V1.0.0.29

- added getRealCOMPort using serial port GUID in msysFindComPort
- getCOMPort still uses port GUID



git-svn-id: svn://localhost/SelfServiceCommon/trunk@908 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Interfaces/inc/systools/mSysFindComPort.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/SelfServiceCommon/Interfaces/inc/systools/mSysFindComPort.h b/SelfServiceCommon/Interfaces/inc/systools/mSysFindComPort.h
index b9628bd3..b393b967 100644
--- a/SelfServiceCommon/Interfaces/inc/systools/mSysFindComPort.h
+++ b/SelfServiceCommon/Interfaces/inc/systools/mSysFindComPort.h
@@ -20,12 +20,15 @@ namespace mSysCOMPort
   // returns empty string if not matching device found
   std::string getCOMPort(const std::string& deviceTypePrefix, const std::string& deviceIdPrefix, const std::string& instanceIdPrefix,
     mSysUSBDevice::Search search, MASSAI::trace::TraceInterface& trc);
+  std::string getRealCOMPort(const std::string& deviceTypePrefix, const std::string& deviceIdPrefix, const std::string& instanceIdPrefix,
+    mSysUSBDevice::Search search, MASSAI::trace::TraceInterface& trc);
 
   // id: example: "USB/VID_10C4&PID_EA60/006311BE" or "VID_10C4&PID_EA60"
   //
   // returns COM port of first matching device
   // returns empty string if not matching device found
   std::string getCOMPort(const std::string& idPart, mSysUSBDevice::Search search, MASSAI::trace::TraceInterface& trc);
+  std::string getRealCOMPort(const std::string& idPart, mSysUSBDevice::Search search, MASSAI::trace::TraceInterface& trc);
 }
 
 #endif
-- 
2.41.0.windows.1

